tests/chains/Cosmos/THORChain/SignerTests.cpp

```diff
@@ -172,6 +174,79 @@ TEST(THORChainSigner, SignTx_MsgDeposit) {
     EXPECT_EQ(output.error_message(), "");
 }
 
+TEST(THORChainSigner, SignTx_Maya_Protobuf_7E480F) {
```

Comment:
Could you please move these tests to `tests/chains/Cosmos/MAYAChain/SignerTests.cpp`?

Reply:
Moved to `tests/chains/Cosmos/MAYAChain/SignerTests.cpp`